spec/helpers/alert_notifier_helper_spec.rb

```diff
@@ -147,4 +147,52 @@
       end
     end
   end
+
+  describe "#subject" do
+    let(:alert) { create(:alert, address: "123 Sample St") }
+    let(:application) do
+      mock_model(Application, address: "Bar Street",
+      description: "Alterations & additions", council_reference: "007",
+      location: double("Location", lat: 1.0, lng: 2.0))
+    end
+    let(:comment) { create(:comment, application: application) }
+
+    context "with an application" do
+      subject { helper.subject(alert, [application], []) }
+      it { should eql "1 new planning application near 123 Sample St" }
+    end
+
+    context "with a comment" do
+      subject { helper.subject(alert, [], [comment]) }
```

Comment:
>Ruby 2.0 introduced first-class support for keyword arguments...

Wow, I had no idea! That's super cool. I've still been using options hashes.

I don't think I'm going to do this as part of this PR though. Mainly because:

>Unfortunately, Ruby 2.0 doesn’t have built-in support for _required_ keyword arguments.

It can be a cool improvement for someone to make down the line :)